src/main/java/org/jvnet/hudson/test/LoggerRule.java

```diff
@@ -162,4 +166,97 @@ protected void after() {
         }
     }
 
+    /**
+     * Creates a {@link Matcher} that matches if the {@link LoggerRule} has a {@link LogRecord} at
+     * the specified {@link Level}, with a message matching the specified matcher, and with a
+     * {@link Throwable} matching the specified matcher.
+     *
+     * @param level The {@link Level} of the {@link LoggerRule} to match. Pass {@code null} to match any {@link Level}.
+     * @param message the matcher to match against {@link LogRecord#getMessage}
+     * @param thrown the matcher to match against {@link LogRecord#getThrown()}
```

Comment:
or null to match against any throwable, or none (as opposed to passing a matcher that matches no throwable)